src/templates/typescript.ts

```diff
@@ -0,0 +1,139 @@
+import type { Template } from '../lib/shared';
+import { Directory } from '../lib/tree/Directory';
+
+export const template: Template = {
+	main: 'dist/{name}.js',
+	scripts: {
+		lint: 'eslint src --ext ts --fix',
+		format: 'prettier --write src/**/*.ts',
```

Comment:
```suggestion
		format: 'prettier --write \\"src/**/*.ts\\"',
```

Or whatever outputs `\"` in the JSON. Bit of a wildcard quirk but (see https://prettier.io/docs/en/cli.html) but this way it is ensured that the wildcards can be resolved by Prettier's globber instead of the OS. Thus providing a proper cross-platform script.

So this final script in `package.json` should be:
```json
{
  "format": "prettier --write \"src/**/*.ts\""
}
```